sites/perioimplantadvisory.com/server/styles/index.scss

```diff
@@ -14,8 +14,6 @@ $theme-site-navbar-primary-bg-color: $primary;
 $theme-site-navbar-menu-hover-background-color: lighten($theme-site-navbar-primary-bg-color, 80%);
 $theme-site-navbar-secondary-type: light;
 
-$theme-item-description-max-lines: 5;
-
 $theme-site-header-breakpoints: (
   hide-primary: 704px,
   hide-secondary: 507px,
```

Comment:
This file is missing the core scss import from the shared package. That should be included and the other imports removed